src/components/position-at-box-shape-border.js

```diff
@@ -1,3 +1,5 @@
+import { getBox } from "../utils/auto-box-collider.js";
```

Comment:
This file should really be cleaned up, but its localized and I would rather get this PR out and fix this in a followup one.